docs/outlook/authenticate-a-user-with-an-identity-token.md

```diff
@@ -1,13 +1,16 @@
 ---
 title: Authenticate a user with an identity token in an add-in
 description: Learn about using the identity token provided by an Outlook add-in to implement SSO with your service.
-ms.date: 10/12/2021
+ms.date: 03/21/2024
 ms.topic: how-to
 ms.localizationpriority: medium
 ---
 
 # Authenticate a user with an identity token for Exchange
 
+> [!IMPORTANT]
+> Legacy Exchange user identity tokens and callback tokens will be turned off for all Exchange Online tenants in October 2024 as part of [Microsoft’s Secure Future Initiative](https://blogs.microsoft.com/on-the-issues/2023/11/02/secure-future-initiative-sfi-cybersecurity-cyberattacks/), which gives organizations the tools needed to respond to the current threat landscape. Exchange user identity tokens will still work for Exchange on premise. Nested app authentication is the recommended approach for tokens going forward. For more information see our [blog post about nested app authentication and legacy Exchange tokens](https://aka.ms/NAApreviewblog).
```

Comment:
```suggestion
> Legacy Exchange user identity tokens and callback tokens will be turned off for all Exchange Online tenants in October 2024 as part of [Microsoft’s Secure Future Initiative](https://blogs.microsoft.com/on-the-issues/2023/11/02/secure-future-initiative-sfi-cybersecurity-cyberattacks/), which gives organizations the tools needed to respond to the current threat landscape. Exchange user identity tokens will still work for Exchange on-premises. Nested app authentication is the recommended approach for tokens going forward. For more information, see our [blog post about nested app authentication and legacy Exchange tokens](https://aka.ms/NAApreviewblog).
```

From style guide: _**_on-premises_**_ Hyphenate in all positions. Premises is plural. Don't use on-premise, off-premise.